ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineExecutePayload.java

```diff
@@ -75,6 +80,14 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
     final ExecutionEngineNewBlockParameter blockParam =
         requestContext.getRequiredParameter(0, ExecutionEngineNewBlockParameter.class);
 
+    Object reqId = requestContext.getRequest().getId();
```

Comment:
Don't we know the object type here  ?

Reply:
We don't.  JsonRpcRequestId is a plain object for some reason 🤷